project/Dependencies.scala

```diff
@@ -72,7 +72,7 @@ object Dependencies {
   val libPhoneNumber = "com.googlecode.libphonenumber" % "libphonenumber" % "7.2.4"
   val logback = "net.logstash.logback" % "logstash-logback-encoder" % "4.6"
   val kinesisLogbackAppender = "com.gu" % "kinesis-logback-appender" % "1.3.0"
-  val targetingClient = "com.gu" %% "targeting-client-play24" % "0.6.0"
+  val targetingClient = "com.gu" %% "targeting-client-play24" % "0.7.0"
```

Comment:
when you bump the version, also link to the relevant PRs in targeting-client so frontend devs can easily review your changes there too